src/vs/editor/contrib/codeAction/browser/codeActionWidgetContribution.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { editorConfigurationBaseNode } from 'vs/editor/common/config/editorConfigurationSchema';
+import * as nls from 'vs/nls';
+import { ConfigurationScope, Extensions, IConfigurationRegistry } from 'vs/platform/configuration/common/configurationRegistry';
+import { Registry } from 'vs/platform/registry/common/platform';
+
+Registry.as<IConfigurationRegistry>(Extensions.Configuration).registerConfiguration({
+	...editorConfigurationBaseNode,
+	properties: {
+		'editor.experimental.codeActionWidget.enabled': {
+			type: 'boolean',
+			scope: ConfigurationScope.LANGUAGE_OVERRIDABLE,
+			description: nls.localize('codeActionWidget', "Enable/disable opening the experimental Code Action Widget."),
```

Comment:
I think this setting name and description are a little confusing. Maybe give the setting a name such as `editor.experimental.useCustomCodeActionMenu` and explain that enabling this switches how the code action menu is rendered (it doesn't turn anything)

See `window.titleBarStyle` for a somewhat similar case